docs/getting_started.md

```diff
@@ -10,7 +10,20 @@ If you want to start your own installation of Decidim, you don't need to clone t
 
 ## Creating your Decidim app
 
-### Using Docker [experimental]
+### A. Using installation script [experimental]
+
+> *Please note that this is **experimental***
+
+We've made an script for Ubuntu 16.04 LTS and macos sierra 10.2. It's a BETA and as such you should be aware that this could break your environment (if you have any). It'll install rbenv, postgresql, nodejs and install decidim on this directory. It should take 15 minutes depending on your network connection.
+
+```
+wget http://get.decidim.org -O install_decidim.bash
+bash install_decidim.bash
+```
+
+Read more about the [installation script](https://github.com/alabs/decidim-install).
+
+### B. Using Docker [experimental]
```

Comment:
This is actually not that experimental (we're using it in several production deployments without issues it so far), but I would leave it this way until we document everything under https://github.com/decidim/docker .